host/4/dotnet-build.yml

```diff
@@ -101,11 +101,21 @@ jobs:
           dockerfilePath: "host/4/mariner/dotnet/dotnet-isolated/"
           dockerImagePath: "dotnet-isolated"
           dockerImageName: "-dotnet-isolated8.0-mariner"
-        _bookwork-net-6:
+        _bookworm-net-6:
           dotnetVersion: "dotnet6"
           dockerfilePath: "host/4/bookworm/dotnet/"
           dockerImagePath: "dotnet"
           dockerImageName: "-dotnet6-bookworm"
+        _bookworm-net-8:
+          dotnetVersion: "dotnet8"
+          dockerfilePath: "host/4/bookworm/dotnet/"
+          dockerImagePath: "dotnet"
+          dockerImageName: "-dotnet8"
+        _appservice-net-8:
+          dotnetVersion: "dotnet8"
+          dockerfilePath: "host/4/bookworm/dotnet/"
+          dockerImagePath: "dotnet"
+          dockerImageName: "-dotnet8-appservice"
```

Comment:
I don't think that this is building your dotnet appservice image. I think its building the base image with no appservice stuff

Reply:
ah ok how would I build the app service image?

Reply:
Target the appservice dockerfile.  I think that its probably the imagePath here. Check how it is being used in the build step

Reply:
done!